Plugins.Import/Csv/GenericCsvParser.cs

```diff
@@ -394,11 +387,13 @@ record = (IDictionary<string, object>) csv.GetRecord<dynamic>();
 
                     while (csv.Read())
                     {
-                        if (csv.Parser.FieldReader.IsBufferEmpty)
+                        //TODO: Fix this
```

Comment:
Is this a temporary fix, or planned fixed for this release?

Reply:
This should be tested with different datasets to see if the CsvHelper now nicely handles reading when reaching the end of the (temporary) buffer.

Reply:
We will include this in our internal build and test using various datasets.